### PR TITLE
Added hook-lark.py and the required lines in setup.py

### PR DESCRIPTION
**Pull request by MegaIng**
Note: tests could still be added, and someone else should check if this correct.

### CODE REVIEW

File: lark/__pyinstaller/hook-lark.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2017-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
```
Comment (Legorooj) [inline lark/__pyinstaller/hook-lark.py:10]: Can you change the copyright header? We only put these on hooks in our repo to stop someone suing us. I'd change to the standard lark header.

Comment (erezsh) [reply lark/__pyinstaller/hook-lark.py:10]: We can probably just remove it. Maybe I'm naive, but I'm not worried about being sued, and Lark's MIT license already covers this file.

Comment (Legorooj) [reply lark/__pyinstaller/hook-lark.py:10]: @erezsh your repo, your choice. You're pretty safe - we don't actually need to have the copyright header, but it's that little bit more security in a hypothetical case.

Comment (MegaIng) [reply lark/__pyinstaller/hook-lark.py:10]: I copied the file over from the other PR. I didn't notice the header was still in there. I would say remove is the best option.

**Pull request merged**